crates/turborepo-lib/src/turbo_json/loader.rs

```diff
@@ -356,9 +356,9 @@ mod test {
     use crate::{task_graph::TaskDefinition, turbo_json::CONFIG_FILE};
 
     #[test_case(r"{}", TurboJson::default() ; "empty")]
-    #[test_case(r#"{ "globalDependencies": ["tsconfig.json", "jest.config.js"] }"#,
+    #[test_case(r#"{ "globalDependencies": ["tsconfig.json", "jest.config.ts"] }"#,
```

Comment:
I was unsure about this at first, but it seems to be happy.  please double-check.